lib/modules/nightMode.js

```diff
@@ -121,17 +128,20 @@ module.beforeLoad = () => {
 		removeNightMode();
 	}
 
-	toggle = new CustomToggles.Toggle('nightMode', i18n('nightModeToggleText'), module.options.nightModeOn.value);
-	toggle.onToggle(type => {
-		if (type === 'manual') overrideNightMode();
-		Options.set(module, 'nightModeOn', toggle.enabled);
-	});
-	toggle.onStateChange(() => {
-		if (toggle.enabled) addNightMode();
-		else removeNightMode();
-		StyleTweaks.toggleSubredditStyleIfNecessary();
-	});
-	toggle.addCLI('ns');
+	// Add the night mode toggle in the gear menu *only for the old design*
+	if (isAppType("r2")) {
+		toggle = new CustomToggles.Toggle('nightMode', i18n('nightModeToggleText'), module.options.nightModeOn.value);
+		toggle.onToggle(type => {
+			if (type === 'manual') overrideNightMode();
+			Options.set(module, 'nightModeOn', toggle.enabled);
+		});
+		toggle.onStateChange(() => {
+			if (toggle.enabled) addNightMode();
+			else removeNightMode();
+			StyleTweaks.toggleSubredditStyleIfNecessary();
+		});
+		toggle.addCLI('ns');
+	}
```

Comment:
FUTURE: Add a "on nightmode toggled" watcher for the redesign. https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/cookies/onChanged